src/Analyzer/WebJobs.VSAnalyzer.Core/packages.config

```diff
@@ -69,7 +69,7 @@
   <package id="System.Reflection.Primitives" version="4.3.0" targetFramework="net461" />
   <package id="System.Resources.ResourceManager" version="4.3.0" targetFramework="net461" />
   <package id="System.Runtime" version="4.3.0" targetFramework="net461" />
-  <package id="System.Runtime.CompilerServices.Unsafe" version="4.4.0" targetFramework="net461" />
+  <package id="System.Runtime.CompilerServices.Unsafe" version="5.0.0" targetFramework="net461" />
```

Comment:
Discussing offline. But the major bump here is concerning. We need to evaluate the serialization needs for the new type to make a decision on how to proceed.

Reply:
Deserialization is not a use case we need to support, reverting those changes